### ISSUE TITLE
How does this optimizer perform in you tasks?

### ISSUE THREAD

**Issue by uYu** [lucidrains/lion-pytorch#3]
How does this optimizer perform? What's your lr and batch size when compared with AdamW?
I utilize the parameters almost the same as AdamW in my NLP task, which performs well.

**Comment by lucidrains** [comment]
@uYu i only spent about 2-3 hours running experiments yesterday before moving on to other work

the divide by learning rate by 3 rule worked well, better than Adam. then i tried dividing by 10 and it didn't work as well. overall, it doesn't diverge, that is for sure. whether it can truly beat Adam on every task, i think i'll just leave it up to the crowd to discover

**Comment by lucidrains** [comment]
@uYu you should run some experiments, share your results in the discussion forums
